src/GovUk.Education.ExploreEducationStatistics.Admin/Models/Api/DataBlockViewModel.cs

```diff
@@ -21,8 +23,12 @@ public class DataBlockViewModel
 
         public List<IContentBlockChart> Charts { get; set; }
 
+        public int Order { get; set; }
```

Comment:
Should the Order field not be a field on IContentBlockViewModel?

Reply:
Oh I see, it's an interface, gotcha.  Maybe there's scope for an abstract base class at some point  in the future.

Reply:
As above, I agree but trying to minimise changes at the moment because I'm wary of the amendments PR.